frontend/src/actions/cartAction.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
```

Comment:
Codacy has a fix for the issue: [Replace `'axios'` with `"axios"`](https://app.codacy.com/gh/Durotemy/e-commerce/pullRequest?prid=9481957)
```suggestion
import axios from "axios";
```